plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionConfigProvider.java

```diff
@@ -23,15 +23,20 @@
 
     private FissionFunctionConfigProvider() {}
 
+    /**
+     * Fetetches env var 'FISSION_FUNCTION_BASE_URL' and returns it as a string, if it is not set will throw an error
```

Comment:
```suggestion
     * Fetches env var 'FISSION_FUNCTION_BASE_URL' and returns it as a string, if it is not set will throw an error
```
You could add a @throws annotation to say "@throws IOException if FISSION_FUNCTION_BASE_URL env var is not set" (or something)